.gitignore

```diff
@@ -20,3 +20,5 @@ node_modules
 
 # .cache directories
 **/.cache
+
+.env
```

Comment:
How do you load this .env file ?

Generally speaking I'm against the use of dotenv...let's use `envalid` and in the doc use `export` in the terminal to load the env variables.

Reply:
@riccardoio The method to load the .env file is clearly mentioned in the readme. 

Also truelayer client sdk uses dotenv to load the .env file by default.

Reply:
1) what is truelayer client sdk ? in the truelayer-client-javascript lib we don't use dotenv: https://github.com/TrueLayer/truelayer-client-javascript

2) I know but it's because you have added it...what I suggest it's that instead to use dotenv and the .env file, we tell the user to declare the env variables with:
`export NAMEVAR=VALUE`

The reason behind this is that I don't trust users, and I want avoid that they paste their secrets in the .env or .env.sample and then commit by error(ex: they change something in the gitignore)

Btw as I said it's not a big deal for this project, and this is the reason I didn't 👎  the PR. If you want go with the dotenv lib, go with it 👍

Reply:
@riccardoio My bad - I missed that. I added `dotenv` back to the sample app. 

> The reason behind this is that I don't trust users, and I want avoid that they paste their secrets in the .env or .env.sample and then commit by error(ex: they change something in the gitignore)
Why will out users commit to our codebase?

Reply:
They won't commit to our codebase, but they can copy and paste, forget the gitignore file and commit in another github repo.

Said this, I will approve the PR because it's not a big deal, and decide what you  think it's best in this case 👍